install.sh

```diff
@@ -69,16 +71,57 @@ case "$mirror" in
 		;;
 esac
 
-# docker-ce-rootless-extras is packaged since Docker 20.10.0
-has_rootless_extras="1"
-if echo "$VERSION" | grep -q '^1'; then
-	has_rootless_extras=
-fi
-
 command_exists() {
 	command -v "$@" > /dev/null 2>&1
 }
 
+# is_at_least_version checks if the version specified in $VERSION is at least
```

Comment:
oh! forgot to update this comment (renamed the function at some point); let me fix

Reply:
done 👍